--- origsrc/espeak-1.46.01-source/src/Makefile	2011-11-22 15:46:08.000000000 -0600
+++ src/espeak-1.46.01-source/src/Makefile	2011-11-27 03:48:45.112118300 -0600
@@ -110,10 +110,14 @@ $(BIN2_NAME): $(OBJS3) $(LIB_NAME)
 
 
 x_%.o: %.cpp
-	$(CXX) $(CXXFLAGS) $(USE_AUDIO) -D PATH_ESPEAK_DATA=\"$(DATADIR)\" -Wall -fpic -fvisibility=hidden -pedantic \
+	$(CXX) $(CXXFLAGS) $(USE_AUDIO) -D PATH_ESPEAK_DATA=\"$(DATADIR)\" -Wall -pedantic \
 	-I. -D LIBRARY -c -fno-exceptions  $<  -o x_$*.o
 
-$(LIB_NAME): $(OBJS2)
+libespeak.dll.a: $(OBJS2)
+	$(CXX) $(LDFLAGS) -shared -Wl,--out-implib,$@ -o cygespeak-$(LIB_VERSION).dll \
+	$(OBJS2) $(LIBS2)
+
+libespeak.so: $(OBJS2)
 	$(CXX) $(LDFLAGS) -shared $(SONAME_OPT)$(LIB_NAME).$(LIB_VERSION) -o $@ \
 	$(OBJS2) $(LIBS2)
 	mv $(LIB_NAME) $(LIB_NAME).$(LIBTAG)
--- origsrc/espeak-1.46.01-source/src/speech.h	2011-11-22 14:47:46.000000000 -0600
+++ src/espeak-1.46.01-source/src/speech.h	2011-11-27 03:32:52.201615000 -0600
@@ -42,7 +42,9 @@
 //#define USE_PORTAUDIO
 //#define USE_PULSEAUDIO
 #define USE_NANOSLEEP
+#ifndef __cdecl
 #define __cdecl 
+#endif
 //#define ESPEAK_API  extern "C"
 
 #ifdef LIBRARY
